client/src/components/DataForm/index.jsx

```diff
@@ -1,15 +1,54 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Field } from 'formik';
-import { Button, Checkbox, FormControlLabel, Typography, FormHelperText } from '@material-ui/core';
+import {
+  Button,
+  Checkbox,
+  FormControlLabel,
+  Typography,
+  FormHelperText,
+  Select,
+  MenuItem,
+  InputLabel,
+  FormControl
+} from '@material-ui/core';
 
 import InputDataTable from '../InputDataTable';
 import NumOfRandomJobsInput from '../NumOfRandomJobsInput';
+import problem10jobs from '../../config/problems/problem10jobs.json';
+import problem20jobs from '../../config/problems/problem20jobs.json';
+import problem35jobs from '../../config/problems/problem35jobs.json';
 
 import styles from './styles.module.scss';
 
+const predefinedProblems = [
+  {
+    jobs: problem10jobs,
+    name: 'problem10jobs.json'
+  },
+  {
+    jobs: problem20jobs,
+    name: 'problem20jobs.json'
+  },
+  {
+    jobs: problem35jobs,
+    name: 'problem35jobs.json'
+  }
+];
+
 const DataForm = ({ jobs, isRandom, setJobs }) => {
   const [fileError, setFileError] = useState(null);
+  const [predefinedProblem, setPredefinedProblem] = useState('');
```

Comment:
why useState('')?

Reply:
This value is used by Select component when we don't want to have any option selected